dev_guide/builds.adoc

```diff
@@ -519,9 +531,42 @@ For example, defining a custom HTTP proxy to be used during build:
 ----
 ====
 
+[[source-code]]
+
+== Git Repository Source Options
+
+The source code location is one of the required parameters for the
```

Comment:
it's not required any more, since you can have a buildconfig that just has a Dockerfile now: https://github.com/openshift/origin/blob/master/pkg/build/api/validation/validation.go#L179-L193


(also in the very near future when this doc gets updated to reflect binary builds, source will be completely optional for those builds)